app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="inaka.lewisdemo"
+    android:versionCode="@android:integer/config_longAnimTime">
+
+    <application
```

Comment:
According to **Lewis**:
> **Id**: AllowBackup **Severity**: Warning **Category**: Security.
  **Message**: On SDK version 23 and up, your app data will be automatically backed up and restored on app install. Consider adding the attribute `android:fullBackupContent` to specify an `@xml` resource which configures which files to backup. More info: https://developer.android.com/preview/backup/index.html.
  **Explanation**: The `allowBackup` attribute determines if an application's data can be backed up and restored. It is documented at http://developer.android.com/reference/android/R.attr.html#allowBackup  By default, this flag is set to `true`. When this flag is set to `true`, application data can be backed up and restored by the user using `adb backup` and `adb restore`.  This may have security consequences for an application. `adb backup` allows users who have enabled USB debugging to copy application data off of the device. Once backed up, all application data can be read by the user. `adb restore` allows creation of application data from a source specified by the user. Following a restore, applications should not assume that the data, file permissions, and directory permissions were created by the application itself.  Setting `allowBackup="false"` opts an application out of both backup and restore.  To fix this warning, decide whether your application should support backup, and explicitly set `android:allowBackup=(true|false)"`.  If not set to false, and if targeting API 23 or later, lint will also warn that you should set `android:fullBackupContent` to configure auto backup.